src/main/java/com/google/devtools/build/lib/runtime/commands/CleanCommand.java

```diff
@@ -178,8 +201,12 @@ private void actuallyClean(CommandEnvironment env,
           env.getWorkspaceName(), "execroot/" + env.getWorkspaceName() }) {
         Path child = outputBase.getRelative(directory);
         if (child.exists()) {
-          LOG.finest("Cleaning " + child);
-          FileSystemUtils.deleteTreesBelow(child);
+          LOG.finest("Cleaning " + child + (cleanOptions.async ? " asynchronously..." : ""));
+          if (cleanOptions.async) {
+            asyncClean(env, child, "Child");
```

Comment:
Ok. How about "Output tree"? In the expunge case, we write "Output base", which also includes a bunch of other things. I suppose the distinction is a bit subtle, but I don't see any good way to explain it with two words.